config/v1/types_scheduling.go

```diff
@@ -28,6 +28,9 @@ type SchedulingSpec struct {
 }
 
 type SchedulingStatus struct {
+	// defaultNodeSelector helps set the cluster-wide default node selector to restrict pod placement to specific nodes.
```

Comment:
1. doc needs examples.
2. describe empty behavior
3. link to doc explaining what this is?

Reply:
and explain how a namespace can opt-out of these.

Reply:
Sure, I will add example.